src/components/RadioList/RadioButton.tsx

```diff
@@ -0,0 +1,35 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { useMemoizedId } from '@hooks/useMemoizedId';
+import { checkIfContainInteractiveElements } from '@utilities/elements';
+import React, { useRef } from 'react';
+
+export enum RadioButtonEmphasis {
+    Default = 'Default',
+    Weak = 'Weak',
+}
+
+export type RadioButtonProps = {
+    value: string;
+    disabled?: boolean;
+    hideLabel?: boolean;
+    emphasis?: RadioButtonEmphasis;
+    helperText?: string;
+};
```

Comment:
I believe the Tooltip is missing on the RadioButton, so the Required state with the little red *

Reply:
I did a quick check and it seems like the keyboard navigation is not working properly. Also a few roles, states & properties are missing, see this [W3C documentation](https://www.w3.org/WAI/ARIA/apg/patterns/radio/). Also I was wondering if we should call it RadioGroup instead of RadioList? Seems more common